Automattic/wp-calypso
# Update Cloud empty purchases screen to RNA design.

#### Changes proposed in this Pull Request

In the Jetpack Cloud Purchases section, when user does not have any purchases, this PR updates the empty purchases screen to Jetpack RNA design. See screenshot)

Figma designs: 1YO0iEGd6N9rffM0P7h8Ws-fi-2222%3A10029
P2: p1HpG7-fg7-p2#comment-53338
Asana task: 1164141197617539-as-1202050456445922

![Screenshot on 2022-03-30 at 11-52-42](https://user-images.githubusercontent.com/11078128/160877858-bb97933e-3530-4519-a56f-502e1074c445.png)


#### Testing instructions



- Checkout this PR and spin up Calypso blue (`yarn start`)  and Calypso green (`yarn start-jetpack-cloud-p`) concurrently.
- You'll need a connected Jetpack site with no purchases. If you don't have one, create a new site with JN, connect Jetpack, and Select Free plan.
- Visit `http://jetpack.cloud.localhost:3001/purchases/subscriptions/:site` (Replace `:site` in the url with your site slug.)
- Verify the the "empty purchases" screen looks good and per design (See figma link above).
- Verify the CTA points to `/pricing/:site`.
- In the Purchases top Nav, verify the first link renamed from 'Active upgrades' to 'My Plan', Verify all the nav links work correctly.
- Verify everything looks ok on mobile view.

Mobile:

https://user-images.githubusercontent.com/11078128/161114407-70298628-f269-4a00-bc3a-fe2e8cf702d2.mp4
